.gitignore

```diff
@@ -120,3 +120,9 @@ compile_commands.json
 via*.json
 /keyboards/**/keymaps/via/*
 /keyboards/**/keymaps/vial/*
+
+
+
+# ignore my keymap stuff
+/keyboards/crkbd/keymaps/
+/keyboards/sofle/keymaps/
```

Comment:
```suggestion
```

Please revert this change.

Reply:
Does this commit work, or do I need to do something more specific to remove them properly.

Reply:
If the file is still shown in the `Files changed` tab, then it still needs to be reverted.